app/models/active_operator/operation.rb

```diff
@@ -28,17 +33,21 @@ def process!
     end
   end
 
-  def fail!
-    update!(failed_at: Time.current)
+  def errored!
+    return false unless persisted?
```

Comment:
I feel like it'd be good to save an errored record, just in case. Otherwise I don't know how valuable the errored state is.

Reply:
Yeah, I was toying with this, in case the exception being rescued was due to the `update!` on the operation (similar to the situation above: `Location.new.geocoding.perform`). But maybe it's fine just to have it call `update!` here as well as it will just raise again.

Reply:
I think it'll be easier to reason about from the user's perspective.